pkg/storage/stores/tsdb/head_manager.go

```diff
@@ -170,8 +170,11 @@ func (m *HeadManager) loop() {
 	for {
 		select {
 		case <-ticker.C:
+			m.metrics.tsdbHeadRotationsTotal.Inc()
```

Comment:
This one is always incremented even if it fails below. I think it should only be incremented when we succeed.

Reply:
yeah, currently all these are also tracking attempts. I can move these around in the refactoring work that we discussed